src/status_im/contexts/wallet/swap/setup_swap/view.cljs

```diff
@@ -98,11 +98,14 @@
                                                   (hex/normalize-hex
                                                    approval-amount-required))
                                                  pay-token-decimals)))
-        pay-input-error?                 (and (not (string/blank? pay-input-amount))
-                                              (money/greater-than
-                                               (money/bignumber pay-input-num-value)
-                                               (money/bignumber
-                                                pay-token-balance-selected-chain)))
+        pay-input-error?                 (or (and (not (string/blank? pay-input-amount))
```

Comment:
for some of these value that do some calculation, like `pay-token-fiat-value`, we should use `use-memo` to memoized value and avoid recalculation between  re-renders.